pkgs/applications/video/mpv/scripts/eisa01.nix

```
@@ -54,6 +54,14 @@ lib.recurseIntoAttrs (
       };
     };
 
+    simplebookmark = {
+      scriptPath = "scripts/SimpleBookmark.lua";
+      meta = {
+        description = "Automatically or manually skip opening, intro, outro, and preview";
```

```suggestion
        description = "Simple bookmarks script based on assigning keys";
```
Or any other description that is accurate